### PR TITLE
PHEE-457 : Fixed the payload while sending auth request to mockpaymentschema

### PR DESCRIPTION
**Pull request by danishjamal104**
## Description

* Added a fallback logic to populate the `amount` field in mock payment schema authorization API call.

[PHEE-457](https://fynarfin.atlassian.net/browse/PHEE-457)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Design related bullet points or design document link related to this PR added in the description above.

- [x] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [x] Create/update unit or integration tests for verifying the changes made.

- [x] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [x] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

[PHEE-457]: https://fynarfin.atlassian.net/browse/PHEE-457?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -108,7 +108,7 @@ dependencies {
     // miscellaneous dependency
     implementation 'com.google.code.gson:gson:2.8.9'
     implementation 'org.json:json:20210307'
-    implementation 'org.mifos:ph-ee-connector-common:1.6.0'
+    implementation 'org.mifos:ph-ee-connector-common:1.6.1-rc.2'
```
Comment (fynmanoj) [inline build.gradle:111]: you can use v1.7.0  snapshot here

File: src/main/java/org/mifos/processor/bulk/zeebe/worker/AuthorizationWorker.java [hunk 2]
```diff
@@ -74,6 +76,9 @@ public void setup() {
 
     private HttpStatus invokeBatchAuthorizationApi(String batchId, AuthorizationRequest requestPayload, String clientCorrelationId) throws JsonProcessingException {
         logger.info("Calling auth API");
+        if (requestPayload.getAmount() == null || requestPayload.getAmount().isEmpty()) {
```
Comment (fynmanoj) [inline src/main/java/org/mifos/processor/bulk/zeebe/worker/AuthorizationWorker.java:79]: can we use a  util function to check the empty condition( eg: SrtingUtils.isNotBlank)

**Review by fynmanoj — CHANGES_REQUESTED** [review]
please update the connector-common version

File: build.gradle (hunk 1 above)
Comment (danishjamal104) [reply build.gradle]: done

File: src/main/java/org/mifos/processor/bulk/zeebe/worker/AuthorizationWorker.java (hunk 2 above)
Comment (danishjamal104) [reply src/main/java/org/mifos/processor/bulk/zeebe/worker/AuthorizationWorker.java]: done

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**